ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -66,6 +67,39 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     Delete Pipeline Server    ${PRJ_TITLE}
     Delete Data Science Project    ${PRJ_TITLE}
 
+Verify Pipeline Metadata Pods Are Not Deployed When Running Pipelines
+    [Documentation]    Verifies that metadata pods are not created when running a data science pipeline,
+    ...         as this feature is currently disabled.
+    [Tags]    Smoke
+    ...       Tier1
+    Create Pipeline Server    dc_name=${DC_NAME}
+    ...    project_title=${PRJ_TITLE}
+    Wait Until Pipeline Server Is Deployed    project_title=${PRJ_TITLE}
+    Import Pipeline    name=${PIPELINE_TEST_NAME}
+    ...    description=${PIPELINE_TEST_DESC}
+    ...    project_title=${PRJ_TITLE}
+    ...    filepath=${PIPELINE_TEST_FILEPATH}
+    Pipeline Should Be Listed    pipeline_name=${PIPELINE_TEST_NAME}
+    ...    pipeline_description=${PIPELINE_TEST_DESC}
+    Capture Page Screenshot
+    ${workflow_name}=    Create Pipeline Run    name=${PIPELINE_TEST_RUN_BASENAME}
+    ...    pipeline_name=${PIPELINE_TEST_NAME}    from_actions_menu=${FALSE}    run_type=Immediate
+    ...    press_cancel=${TRUE}
+    Open Data Science Project Details Page    ${PRJ_TITLE}
+    ${workflow_name}=    Create Pipeline Run    name=${PIPELINE_TEST_RUN_BASENAME}
+    ...    pipeline_name=${PIPELINE_TEST_NAME}    from_actions_menu=${FALSE}    run_type=Immediate
+    Open Data Science Project Details Page    ${PRJ_TITLE}
+    Wait Until Pipeline Last Run Is Started    pipeline_name=${PIPELINE_TEST_NAME}
+    ...    timeout=10s
+    Wait Until Pipeline Last Run Is Finished    pipeline_name=${PIPELINE_TEST_NAME}
+    ...    timeout=180s
+    @{pods} =    Oc Get    kind=Pod    namespace=${PRJ_TITLE}
+    FOR    ${pod}    IN    @{pods}
+        Log    ${pod['metadata']['name']}
+        Should Not Contain    ${pod['metadata']['name']}    ds-pipeline-metadata
+    END
+    Delete Data Science Project    ${PRJ_TITLE}
```

Comment:
I think it should be like this for this and the previous test:
`[Teardown]  Delete Data Science Project    ${PRJ_TITLE}`

If not, I think a failure in the first test will prevent the second one running properly